tests/simulation/test_simulation.py

```diff
@@ -25,16 +25,27 @@
 
 
 class TestSimulation:
+
+    # TODO: Ideally, the db_version should be the (parameterized) db_version
+    # fixture, to test all versions of the database schema. This doesn't work
+    # because of a hack in shutting down the full node, which means you can run
+    # more than one simulations per process.
```

Comment:
pytest does have some helpers for running tests in subprocesses and collecting the results.  Not beautiful, but not hard if you are interested.

https://github.com/pytest-dev/pytest-twisted/blob/87845bd0e1ae2beb66ea9f911043aca075b45e1e/testing/test_basic.py#L568-L588

Reply:
good to know. not for this PR though